data/_demodata/applications/portal/Biosamples.csv

```diff
@@ -0,0 +1,19 @@
+id,collected from individual,included in datasets.resource,included in datasets.name
+SAMEA10450711,Case1F,EGAS00001005702,EGAD00001008392
```

Comment:
is okay. For future we should probably reconsider a long standing discussion on weather we want to treat 'datasets' just like 'resources' having their own identifier (maybe even merge them into one table).